core/src/main/resources/lib/hudson/property-table.js

```diff
@@ -1,21 +1,40 @@
 (function(){
   document.addEventListener("DOMContentLoaded", function() {
-    document.querySelectorAll('.hidden-info .jenkins-button').forEach(function(elem) {
+    document.querySelectorAll('.app-hidden-info-reveal .jenkins-button').forEach(function(elem) {
```

Comment:
new classes are namespaced, `jenkins-` for public API and `app-` for internal classes

Reply:
Where is this specified?

Reply:
I think it’s not written down yet, was discussed somewhere but can’t remember where.

there’s an open issue for it here: https://github.com/jenkinsci/design-library-plugin/issues/58

cc @janfaracik if you have anything to add

Reply:
Nothing written down yet but I'd definitely encourage namespacing, e.g. its great to avoid conflicts with other libraries we might use like Bootstrap.